src/Microsoft.Health.Fhir.Core.UnitTests/Config/EncryptionConfigurationTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Fhir.Core.Configs;
+using Microsoft.Health.Fhir.Tests.Common;
+using Microsoft.Health.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Core.UnitTests.Config
+{
+    [Trait(Traits.OwningTeam, OwningTeam.Fhir)]
+    [Trait(Traits.Category, Categories.Operations)]
+    public sealed class EncryptionConfigurationTests
+    {
+        [Fact]
+        public void GivenAnEncryptionConfiguration_WhenContainsValues_ReturnAsEncryptionConfigurationSet()
+        {
+            EncryptionConfiguration encryption0 = new EncryptionConfiguration()
+            {
+                CustomerManagedKeyEncryption = new CustomerManagedKeyEncryption()
+                {
+                    KeyEncryptionKeyIdentity = new EncryptionKeyIdentity()
+                    {
+                        FederatedClientId = "foo",
+                    },
+                },
+            };
+            Assert.True(encryption0.IsEncryptionSet());
```

Comment:
Is this a valid configuration? I thought the KeyUrl was always required.

Reply:
I need to sync with you about this as well. 
From your tests, are both properties required? Or just the KeyUrl is required? 
What is the usage of the FederalClientId?

Reply:
I made FederatedClientId optional and KeyEncryptionKeyUrl required.